addons/im_livechat/static/tests/embed/feedback_panel_tests.js

```diff
@@ -57,15 +58,11 @@ QUnit.test("Feedback with rating and comment", async (assert) => {
                 assert.step(route);
             }
             if (route === "/im_livechat/chat_post") {
-                if (messageCount === 1) {
-                    assert.strictEqual(
-                        args.message_content,
-                        `Rating: ${RATING_TO_EMOJI[RATING.GOOD]}`
-                    );
-                } else if (messageCount === 2) {
-                    assert.strictEqual(args.message_content, "Rating reason: Good job!");
+                if (messageCount++ === 1) {
+                    assert.ok(args.message_content.includes("Good job!"));
+                    const emojiSrc = `${session.origin}/rating/static/src/img/rating_${RATING.GOOD}.png`;
+                    assert.ok(args.message_content.includes(`src="${emojiSrc}"`));
```

Comment:
I would remove `messageCount`. This looks irrelevant for the test, and also makes reading test harder than it should. (+ I don't like `++` and `===` combined in a single expression 🥲